apps/backend/src/constants.ts

```diff
@@ -1 +1,33 @@
+import {
+  LeaderboardEntryType,
+  OAuthMapping
+} from "@types";
+
 export const FRONTEND_URL = process.env.FRONTEND_URL || "http://localhost:3000";
+
+export const OAUTH_APP_MAPPING: Record<string, OAuthMapping> = {
+  "strava": {
+    client_side_fetching: true,
+    token_url: "https://www.strava.com/api/v3/oauth/token",
+    redirect_uri: "http://connections.cursive.team/oauth/exchange_token&approval_prompt=force&scope=read",
```

Comment:
this should depend on FRONTEND_URL, no? also, does https work or no?

Reply:
Both good points, will change, thanks.

Reply:
Ah I think that's an artifact of https failing on localhost, good catch.